### ISSUE TITLE
Additional connectivity stats to calculate

### ISSUE THREAD

**Issue by dosumis** [VirtualFlyBrain/VFB_connectomics_import#6]
Neuron:neuron

* Proportion of total synaptic output of neuron (before filtering).  TBD - should this include outputs where no reconstructed downstream neuron is known?  We think yes @jefferis @mmc46  please comment.

Neuron:region

* Proportion of total synaptic output from neuron  to neuropil. 
* Proportion of total synaptic input to neuron from neuropil. 

Problem - these add up to more than 100 because nested.  Decision" deal with this crudely using primary and secondary ROI distinction in neuprint.

**Comment by mmc46** [comment]
> Neuron:neuron
> Proportion of total synaptic output of neuron (before filtering). TBD - should this include outputs where no reconstructed downstream neuron is known? We think yes @jefferis @mmc46 please comment.

Yes, I think it should include all outputs, even if the target isn't known.
